images/build-e2e/Dockerfile

```diff
@@ -14,10 +14,6 @@ COPY --from=builder /workspace/out /opt/crc/bin
 # Review this when go 1.16 with embed support
 COPY --from=builder /workspace/test/e2e/features /opt/crc/features
 COPY --from=builder /workspace/test/testdata /opt/crc/testdata
-COPY --from=builder /workspace/test/extended/crc/ux/installer/applescripts /opt/crc/ux/installer/applescripts
-COPY --from=builder /workspace/test/extended/crc/ux/notification/applescripts /opt/crc/ux/notification/applescripts
-COPY --from=builder /workspace/test/extended/crc/ux/tray/applescripts /opt/crc/ux/tray/applescripts
-
```

Comment:
In the commit logs, we should have an explanation around why we are removing those.

Reply:
well those are assets used for ux testing, as we are removing that part we need to remove them from the container image..the commit says _e2e container remove test ux assets_

I can split this into a different PR and move the commits related to remove ux testing from here.